tests/integration/components/public/speaker-list-test.js

```diff
@@ -0,0 +1,18 @@
+import { test } from 'ember-qunit';
+import moduleForComponent from 'open-event-frontend/tests/helpers/component-helper';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('speaker-list', 'Integration | Component | speaker list');
```

Comment:
Should be 
`moduleForComponent('public/speaker-list', 'Integration | Component | public/speaker list');`